sw/host/tests/chip/jtag/BUILD

```diff
@@ -7,9 +7,27 @@ load("@rules_rust//rust:defs.bzl", "rust_binary")
 package(default_visibility = ["//visibility:public"])
 
 rust_binary(
-    name = "jtag",
+    name = "test_openocd",
```

Comment:
Generally, we don't require test harnesses to have `test` in their name, however, if you're going to put `test` into the name, I'd prefer to follow the convention and use `test` as a suffix rather than a prefix (both for rule names and filenames).